test/specs/box.spec.js

```diff
@@ -45,4 +45,27 @@ describe('Box annotation', function() {
       }
     });
   });
+
+  describe('scriptable options', function() {
```

Comment:
Could make sense to have a fixture for box and ellipse where we set a gradient as background color?

Reply:
> > I have no idea what is the purpose of that check. Why do we need to know if the box is visible and not transparent?
> > Removing the backgroundColor stuff there does not make anything fail.
> 
> This specific check was done to avoid to draw the box if border width = 0 and background is transparent. But we can remove it. drawBox function will be called anyway even if it shouldn't draw anything.

Yes, but that is again a breaking change, so lets do that in v2

Reply:
> Could make sense to have a fixture for box and ellipse where we set a gradient as background color?

Added a fixture